app/controllers/reservations_controller.rb

```diff
@@ -35,6 +35,28 @@ def handle_pending_reservation_admin
     end
   end
 
+  # create my reservations for a user
+  def my_reservations
+    return render json: { error: 'You need to loginto access this page' }, status: :unauthorized unless logged_in?
+
+    @reservations = Reservation.find(user_id: 25)
```

Comment:
i think you forget the user_id testing in here 😞 it should be
 ```ruby
@reservations = Reservation.find(user_id: current_user.id)
```

Reply:
@ichala Done